docs/Usage.md

```diff
@@ -95,14 +95,24 @@ IP_PREFIX: "25"                                               # Subnet Mask in C
 DNS_SERVERS: "[8.8.8.8,8.8.4.4]"                              # The dns nameservers for the machines network-config.
 BRIDGE: "vmbr1"                                               # The network bridge device for Proxmox VE VMs
 
-## -- xl nodes-- ## 
-BOOT_VOLUME_DEVICE: "scsi0"                                    # The device used for the boot disk.   
-BOOT_VOLUME_SIZE: "100"                                        # The size of the boot disk in GB.
-NUM_SOCKETS: "2"                                               # The number of sockets for the VMs.
-NUM_CORES: "4"                                                 # The number of cores for the VMs.
-MEMORY_MIB: "8048"                                             # The memory size for the VMs.
-
-EXP_CLUSTER_RESOURCE_SET: "true"                               # This enables the ClusterResourceSet feature that we are using to deploy CNI
+## -- xl nodes -- ##
+BOOT_VOLUME_DEVICE: "scsi0"                                   # The device used for the boot disk.
+BOOT_VOLUME_SIZE: "100"                                       # The size of the boot disk in GB.
+NUM_SOCKETS: "2"                                              # The number of sockets for the VMs.
+NUM_CORES: "4"                                                # The number of cores for the VMs.
+MEMORY_MIB: "8048"                                            # The memory size for the VMs.
+
+## -- loadbalancer nodes -- #
```

Comment:
I would say the docs about load balancing can go to the advanced-setups.md instead.

Reply:
Alright.

Reply:
Done